content/en/docs/tasks/debug-application-cluster/debug-service.md

```diff
@@ -300,7 +300,7 @@ Namespace ("default.svc.cluster.local"), Services in all Namespaces
 ("svc.cluster.local"), and lastly for names in the cluster ("cluster.local").
 Depending on your own install you might have additional records after that (up
 to 6 total).  The cluster suffix is passed into `kubelet` with the
-`--cluster-domain` flag.  We assume that is "cluster.local" in this document,
+`--cluster-domain` flag.  You assume that is "cluster.local" in this document,
 but yours might be different, in which case you should change that in all of
```

Comment:
nit: `we/you` substitution problem, possible edit:
Throughout this document, the cluster suffix equals "cluster.local". Your cluster suffix may be different, in which case you should modify the cluster suffix value in the previous commands.
OR
The cluster suffix, "cluster.local", is passed into the kubelet with the `--cluster-domain` flag. Your cluster suffix may be different, in which case you should modify the cluster suffix value in the previous commands.